Mathlib/RingTheory/Artinian/HopkinsLevitzki.lean

```diff
@@ -0,0 +1,50 @@
+/-
+Copyright (c) 2025 Andrew Yang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Wanyi He, Jiedong Jiang, Jingting Wang, Andrew Yang, Shouxin Zhang
+-/
+import Mathlib.RingTheory.HopkinsLevitzki
+import Mathlib.RingTheory.KrullDimension.Zero
+import Mathlib.RingTheory.Artinian.Ring
+
+/-!
+# Corollaries of Hopkins-Levitzki
+
+In this file, we prove several corollaries of the Hopkins-Levitzki theorem.
+-/
+variable {R : Type*} [CommRing R]
+
+theorem isArtinianRing_iff_krullDimLE_zero [IsNoetherianRing R] :
+    IsArtinianRing R ↔ Ring.KrullDimLE 0 R := by
+  rwa [isArtinianRing_iff_isNoetherianRing_krullDimLE_zero, and_iff_right]
+
+lemma IsArtinianRing.eq_maximalIdeal_of_isPrime [IsArtinianRing R] [IsLocalRing R]
+    (I : Ideal R) [I.IsPrime] : I = IsLocalRing.maximalIdeal R := by
+  have : Ring.KrullDimLE 0 R := by rwa [← isArtinianRing_iff_krullDimLE_zero]
+  exact Ring.KrullDimLE.eq_maximalIdeal_of_isPrime I
```

Comment:
I don't think we want this. `IsArtinianRing R` -> `Ring.KrullDimLE 0 R` should be an instance and this is exactly `Ring.KrullDimLE.eq_maximalIdeal_of_isPrime`.